content/docs/tasks/security/vault-ca/index.md

```diff
@@ -69,53 +65,73 @@ env:
     {{< /text >}}
 
-1.  List Node Agent's pods:
+1.  Deploy the `httpbin` and `sleep` backends:
 
     {{< text bash >}}
-    $ kubectl get pod -n istio-system -l app=nodeagent -o jsonpath={.items..metadata.name}
+    $ kubectl apply -f httpbin-injected.yaml
+    $ kubectl apply -f sleep-injected.yaml
     {{< /text >}}
 
-1.  View each Node Agent's logs. The Node Agent residing on
-the same node as the testing workload will contain Vault related logs.
+## Istio mutual TLS with Vault CA integration
+
+This section demos the use of mutual TLS with the Vault CA integration. In the previous steps,
+you enabled mutual TLS for the Istio mesh and the `httpbin` and `sleep` workloads. These workloads receive
+certificates from a testing Vault CA. When you send a `curl` request from the `sleep` workload
+to the `httpbin` workload, the request goes through a mutual TLS protected channel constructed
+with the certificates the Vault CA issued.
 
     {{< text bash >}}
-    $ kubectl logs -n istio-system THE-POD-NAME-FROM-PREVIOUS-COMMAND
+    $ kubectl exec -it $(kubectl get pod -l app=sleep -o jsonpath='{.items[0].metadata.name}') -c sleep -- curl -s -o /dev/null -w "%{http_code}" httpbin:8000/headers
+    200
     {{< /text >}}
 
-1.  Because in this example, Vault is not configured to accept the Kubernetes JWT
-service account from the `httpbin` workload, you should see that Vault rejects the
-signing requests with the following logs:
-
-    {{< text plain >}}
-    2019-01-16T19:42:19.274291Z     info    SDS gRPC server start, listen "/var/run/sds/uds_path"
-    2019-01-16T19:42:22.015814Z     error   failed to login Vault: Error making API request.
-    URL: PUT https://35.233.249.249:8200/v1/auth/kubernetes/login
-    Code: 500. Errors:
-    * service account name not authorized
-    2019-01-16T19:42:22.016112Z     error   Failed to sign cert for "default": failed to login Vault at https://35.233.249.249:8200: Error making API request.
+1.  Send a `curl` request from the `sleep` workload to the `httpbin` workload.
+    The request succeeds with a `200` response code if it goes through the
+    mutual TLS protected channel constructed with the certificates issued by the Vault CA.
+
+    {{< text bash >}}
```

Comment:
This code block is not rendering with bash syntax highlighting. @geeknoid do you know why?

Reply:
lines 123-127 should come after line 131

Reply:
Done: 1. changed lines 123-127 to come after line 131. 2. rebased to fix the syntax highlighting.